java/README.md

```diff
@@ -0,0 +1,17 @@
+# Java Bindings Development
+
+Install a JDK 17 toolchain.
```

Comment:
Doesn't gradle handle all this for you? I didn't install JDK and was able to run gradle, but see my comment in the main thread.

Reply:
@a10y this is actually a new feature in gradle 8.13 where you can get gradle daemon to self provision the correct jvm toolchain https://docs.gradle.org/current/release-notes.html#toolchain-support

Reply:
i think this is all that was missing [`96240dc` (#2516)](https://github.com/spiraldb/vortex/pull/2516/commits/96240dc53173f6798255ef02c7a4daaf9cea648c)

Reply:
Spark 3 only supports up to 17. Not sure what breaks but it would be simpler if we don’t have to fix people’s Java builds. Spark 4 supports 21 and there’s no lts beyond 21 yet